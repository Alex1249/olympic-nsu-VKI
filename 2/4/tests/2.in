100
8 0
9 4
6 3
2 1
2 2
10 6
6 1
10 2
3 1
1 0
10 4
2 2
9 1
9 2
0 0
3 1
3 0
10 6
4 4
4 0
2 2
1 1
1 0
9 1
9 8
3 0
5 5
5 0
6 2
3 3
7 3
0 0
9 4
7 5
5 3
0 0
6 3
1 0
4 0
3 3
3 3
1 0
10 2
3 1
4 0
6 3
6 5
1 1
0 0
8 4
9 3
1 0
0 0
3 1
4 4
9 1
5 4
5 0
10 10
4 3
5 4
6 4
7 7
6 6
0 0
9 2
3 0
3 1
5 5
2 1
4 0
6 6
6 6
4 3
7 2
2 2
10 10
8 7
9 8
8 7
6 6
6 2
6 4
9 7
0 0
7 4
9 9
8 4
4 2
6 5
5 2
6 4
10 9
5 1
2 2
0 0
4 2
6 5
3 0
9 5
