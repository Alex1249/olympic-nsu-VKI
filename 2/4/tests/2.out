1
126
20
2
1
210
6
45
3
1
210
1
9
36
1
3
1
210
1
1
1
1
1
9
9
1
1
1
15
1
35
1
126
21
10
1
20
1
1
1
1
1
45
3
1
20
6
1
1
70
84
1
1
3
1
9
5
1
1
4
5
15
1
1
1
36
1
3
1
2
1
1
1
4
21
1
1
8
9
8
1
15
15
36
1
35
1
70
6
6
10
15
10
5
1
1
6
6
1
126
